### ISSUE TITLE
Fix problematic keyword

### ISSUE THREAD

**Issue by curleypg** [CodeForCharlotte/VoteSmarterNC#19]
## Description
There exists a keyword in [2017 House Bill 458](https://www2.ncleg.net/BillLookup/2017/H458) that is obviously problematic for our scraper:

```json
BOARDS, EDUCATION, EDUCATION, STATE BOARD 
OF, ELEMENTARY EDUCATION,
```

Why does `EDUCATION` appear twice? It's because the keyword _State Board of Education_ keyword is stored with a comma in the NCGA taxonomy.

## Deliverables
- Fix the bills spider so that keyword `EDUCATION, STATE BOARD OF` is detected so that it's saved as `STATE BOARD OF EDUCATION`

## BONUS POINTS
- In addition to the first deliverable, predict that any keyword that end with ` OF` will follow the same patter of behavior when your scraping. Adjust your spider logic to anticipate these keyword possibilities.

**Comment by derekdkim** [comment]
Hi, would I be able to try and work on this? I'm a beginner and I have never contributed to an open source project before. I'm following the github guide to contributing so I won't be able to do it at lightning speed, but I'll try my best to get it done.

**Comment by curleypg** [comment]
Yes, absolutely, feel free to tackle this problem! I encourage you to 1.) Check out the Scrapy documentation and tutorial (https://doc.scrapy.org/en/latest/) to understand how Scrapy identifies HTML elements and extracts data ffrom webpages. 2.) Clone this repository to your Github account and commit your changes to it. When you're ready make a Pull Request to this repository and we'll review it.

**Comment by derekdkim** [comment]
Thanks. I'm looking forward to learning scrapy.

**Comment by derekdkim** [comment]
I'm a complete beginner so this is the crude solution I came up with to change any keyword that ends with "OF" to its concatenated form with the keyword right before it, then removes the keyword added to the concatenated keyword. I was studying the Scrapy guide but the items formatting went right over my head. From what I understood, it's a list of the information parsed from the API that contains the label "keywords". So I assumed that item["keywords"] is a list. 

I also have no idea where it would be an appropriate write the code in the spider to keep it clean, assuming it's correct.

Sorry about the trouble.

```
for word in keywords:
    if word.endswith("OF"):
        new_keyword = (word + " " + keywords[keywords.index(word) - 1])
        keywords[keywords.index(word)] = new_keyword
        keywords.remove(keywords[keywords.index(new_keyword) - 1])
        print(keywords) #I had this in here to test the method in a separate script
```
